.vscode/settings.json

```diff
@@ -1,5 +1,6 @@
 {
   "editor.defaultFormatter": "esbenp.prettier-vscode",
   "editor.formatOnSave": true,
-  "editor.formatOnPaste": false
+  "editor.formatOnPaste": false,
+  "peacock.color": "#68af4a"
```

Comment:
No Way!

Reply:
I swear there was never any issues with peacock until I fiddled with the settings earlier

Reply:
peacock FTW